vitrina/uapi/__init__.py

```diff
@@ -5,3 +5,31 @@
 class AgentType(TextChoices):
     SPINTA = "SPINTA", "Spinta"
     OTHER = "OTHER", _("Kita")
+
+
+class HttpMethods(TextChoices):
```

Comment:
Abbreviations should still be written in upper-case:

```suggestion
class HTTPMethods(TextChoices):
```